addons/mail/static/tests/helpers/mock_server/models/discuss_channel.js

```diff
@@ -952,7 +953,7 @@ patch(MockServer.prototype, "mail/models/discuss_channel", {
      */
     _mockDiscussChannelWriteImage128(id) {
         this.pyEnv["discuss.channel"].write([id], {
-            avatarCacheKey: moment.utc().format("YYYYMMDDHHmmss"),
+            avatarCacheKey: serializeDateTime(DateTime.utc()).replace(/:|-| /g, ""),
```

Comment:
I would use toFormat here

Reply:
@brboi prefers that we use serializeDateTime and then use replace for this case regarding the previous PR. So ... As you want :D